frontend/src/lib/components/BillingToolbar.js

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import { WarningOutlined } from '@ant-design/icons'
+
+const BillingToolbar = (props) => {
+    const { billingUrl } = props
+
+    return (
+        <>
+            {billingUrl && (
+                <div className="card">
+                    <div className="card-body">
+                        <WarningOutlined className="text-danger" /> Hey! You're about to reach your usage limit for the
```

Comment:
I think this should be changed to "You've reached to usage limit.." as we'll only turn this on once that's done

Reply:
Makes sense! See commit below